src/vcf/assembly_checker.cpp

```diff
@@ -31,6 +31,31 @@ namespace ebi
       std::string get_reference_accession(const std::string& reference_tagged_line);
       std::string get_contig_accession(const std::string& contig_tagged_line);
 
+      bool process_vcf_ref(std::istream & vcf_input,
```

Comment:
we usually order the functions so that the functions appear right after they are used.
if the code is 
```
function a {
    b()
    c()
} 
function b {
    called_from_b()
}
```
we sort them (prototypes too) like this:
```
a()
b()
called_from_b()
c()
```
now that you are cleaning the prototypes in this file, can you order them too?

Reply:
Here is skeleton of functions with ordering. If my interpretation is correct i will change both declarations and definitions in this order.
---------------------------
check_vcf_ref()
 - process_vcf_ref()
 
process_vcf_ref()
 - get_reference_accession()
 - get_contig_accession()
 - process_vcf_records()
 
get_reference_accession()

get_contig_accession()

process_vcf_records()
 - build_record_core()
 - report_telomere_position()
 - get_matching_synonyms_list()
 - report_missing_chromosome()

build_record_core()

report_telomere_position()

get_matching_synonyms_list()
 - report_missing_chromosome()
 - report_multiple_synonym_match()

report_missing_chromosome()
 
report_multiple_synonym_match()